src/coordinator/api/v1/handler/database/create.go

```diff
@@ -54,11 +54,13 @@ const (
 
 	idealDatapointsPerBlock           = 720
 	blockSizeFromExpectedSeriesScalar = idealDatapointsPerBlock * int64(time.Hour)
+	shardMultiplier                   = 32
```

Comment:
I like where you're going with this. In production we've tried to stay around ~200-300 shards per host - that number obvi depends on profile of the h/w and workload. We could shoot for ~200 as the OSS default for now.

The math would look something like:
```
totalNumShards := 64 * numHosts * replicationFactor
closestPowerOfTwo := math.Pow(2, 1+math.Floor(log(totalNumShards)/log(2)))
numShardsPerHost := closestPowerOfTwo / numHosts
```

What do you think?

Reply:
Yep so in my formulas above `numberOfShardsInPlacement * replicationFactor = totalNumShards`.